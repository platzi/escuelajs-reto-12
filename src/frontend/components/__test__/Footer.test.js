import React from 'react';
import { create } from 'react-test-renderer';
import { render, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Footer from '../Footer';

configure({ adapter: new Adapter() });

describe('Footer testing', () => {
  test('Match snapshot', () => {
    const footer = create(<Footer />);
    expect(footer.toJSON()).toMatchSnapshot();
  });

  test('footer has at least 1 division tag', () => {
    const footer = render(<Footer />);
    expect(footer.find('p')).toHaveLength(1);
  });

  test('footer has at least 1 anchor tag', () => {
    const footer = render(<Footer />);
    expect(footer.find('p')).toHaveLength(1);
  });
});
