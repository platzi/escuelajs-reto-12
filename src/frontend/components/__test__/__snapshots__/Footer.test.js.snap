// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer testing Match snapshot 1`] = `
<div>
  <p>
    Todos los Izquierdos Reservados
  </p>
</div>
`;
